src/kibana/plugins/dashboard/styles/main.less

```diff
@@ -72,8 +72,9 @@ dashboard-grid {
 
         .panel-title {
           font-size: inherit;
+          display: inline-block;
```

Comment:
IE10 wasn't playing nice, http://stackoverflow.com/questions/22008135/internet-explorer-10-does-not-apply-flexbox-on-inline-elements.  

I can make the title a header element so it gets block styling instead, but will probably have to override some inherited styles.  Open to any other suggestions.